[tools/shoestring]: Japanese Language Support

ccHarvestasya opened the pull request:
## Problem

- Japanese translation files exist, but all messages are not set.

## Solution

- Japanese translation files were updated.
- Added frequently used but untranslated messages to the translation file.
- Improved wizard display process to support multi-byte characters (double-byte characters) such as Japanese:
  - Changed tab display character width acquisition from `len()` to `get_cwidth()`, which takes into account the actual display width.
  - To correct a problem with button width calculations being off, we have overridden `prompt_toolkit.widgets.Button` and changed the width calculations to support multibyte characters.
  before: ![before](https://pbs.twimg.com/media/GsVsGkkbMAAyOPh?format=png "before")
  after: ![after](https://pbs.twimg.com/media/GsVszuVaMAAFFZN?format=jpg "after")

ccHarvestasya committed:
[tools/shoestring] feat: Japanese language support

ccHarvestasya committed:
[tools/shoestring] fix: Ensure all fragments are appended correctly in ButtonMb

ccHarvestasya committed:
[tools/shoestring] fix: add coverage to .gitignore and create pytest.ini for warning filters

ccHarvestasya committed:
[tools/shoestring] fix: Change the class name for multibyte buttons from ButtonMb to MultibyteButton

ccHarvestasya committed:
[tools/shoestring] fix: Update line numbers in translation files for consistency

ccHarvestasya committed:
[tools/shoestring] fix: Standardize indentation and formatting in test for MultibyteButton

ccHarvestasya committed:
[tools/shoestring] fix: Simplify button initialization in ObligatorySettings by removing redundant variables

ccHarvestasya committed:
[tools/shoestring] fix: Improve Japanese translations for clarity and consistency

Wayonb reviewed (approved):
Looks good. 👍🏾 
There might new strings in the dev branch which also needs to be translated 🙏🏾

ccHarvestasya committed:
[tools/shoestring] fix: add Japanese translation for added bootstrap import

ccHarvestasya commented:
Added Japanese translation of new strings in dev.

ccHarvestasya committed:
[tools/shoestring] revert: .gitignore, pytest.ini
This commit is not consistent with the purpose of the pull request, so it is cancelled

Wayonb reviewed (approved):
👍🏾

ccHarvestasya committed:
fix: improve Japanese translations for clarity and consistency
